spec/stat_tracker_spec.rb

```diff
@@ -60,6 +60,9 @@
       expect(@stat_tracker.best_offense).to eq("Portland Timbers")
     end
 
+    it "#highest_scoring_home_team" do
```

Comment:
This test needs to be written again. I'm not sure if you meant to write the statement like this. It also needs an end